test/addons/rook-toolbox/fetch

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+from drenv import cache
+
+os.chdir(os.path.dirname(__file__))
+path = cache.path("addons/toolbox.yaml")
```

Comment:
Same as rook-cluster, we want to sue the name of the addon - so "addons/rook-toolbox.yaml"